examples/plugins/http-requests/README.md

```diff
@@ -0,0 +1,89 @@
+# HTTP requests plugin
+
+The HTTP requests plugin tracks the number of incoming HTTP requests per process over time using eBPF.
+The UI shows the number of requests as a single value and graph which are updated every second.
+The HTTP requests plugin does not use any application-level instrumentation, it has a negligible performance toll (metrics are obtained in-kernel without any packet copying to userspace).
+
+**Note:** This plugin needs a [recent kernel version with eBPF support](https://github.com/iovisor/bcc/blob/master/INSTALL.md#kernel-configuration).
+It will not compile on current [dlite](https://github.com/nlf/dlite) and boot2docker hosts.
+
+<img src="imgs/http-requests.png" width="800" alt="HTTP requests plugin screenshot" align="center">
+
+If the running plugin has been registered by Scope, you will see it in the list of `PLUGINS` in the bottom right of the UI (see the rectangle in the above figure).
+
+## Tutorial
+
+Follow the next steps to get and run `Scope`, compile and run the HTTP requests plugin, and run a simple webserver to test the plugin.
+
+* Get `Scope`, build it, and run it (it can take a while)
+
+```
+git clone git@github.com:weaveworks/scope.git
+cd scope
+make
+./scope launch
+```
+
+* Run the HTTP requests plugin `cd examples/plugins/http-requests ; make`.
+* In another terminal, run an `nginx` instance `docker run --name http-requests-nginx -p 8080:80 nginx`
+* Run `sh test-http-requests.sh`, this script will open a long-term connection with the web server and then start downloading the main page until Ctrl+c is sent by the user.
+* Go to the Weave Scope UI [http://localhost:4040](http://localhost:4040).
+* Open the `http-requests-nginx` container.
+* Click on `nginx: worker process` in the "Processes" table.
+
+## Report
```

Comment:
I would rename this to something like `plugin internals` and explain what a report is if you use the term.